pipeline/inputs/process-exporter-metrics.md

```diff
@@ -0,0 +1,84 @@
+
+The initial release of Process Exporter Metrics refers a process group level collector to collect metrics available from [the 3rd party implementation of Prometheus Process Exporter](https://github.com/ncabatoff/process-exporter) and we plan to expand them over time if needed.
+
+**Important note:** Metrics collected with Process Exporter Metrics flow through a separate pipeline from logs and current filters do not operate on top of metrics.
+
+This plugin is only supported on Linux based operating systems.
+
+
+## Configuration
+
+| Key                       | Description                                                                            | Default   |
+| ------------------------- | -------------------------------------------------------------------------------------- | --------- |
+| scrape_interval           | The rate at which metrics are collected from the host operating system                 | 5 seconds |
+| path.procfs               | The mount point used to collect process information and metrics                        | /proc/    |
+| process\_include\_pattern | regex to determine which processes are included in the metrics produced by this plugin | It is applied for all process unless explicitly set. Default is `.+`. |
+| process\_exclude\_pattern | regex to determine which units are excluded in the metrics produced by this plugin     | It is not applied unless explicitly set. Default is `NULL`. |
+
+## Getting Started
+
+### Simple Configuration File
+
+In the following configuration file, the input plugin _process\_exporter\_metrics collects _metrics every 2 seconds and exposes them through our [Prometheus Exporter](../outputs/prometheus-exporter.md) output plugin on HTTP/TCP port 2021.
+
+```
+# Process Exporter Metrics + Prometheus Exporter
+# -------------------------------------------
+# The following example collect host metrics on Linux and expose
+# them through a Prometheus HTTP end-point.
+#
+# After starting the service try it with:
+#
+# $ curl http://127.0.0.1:2021/metrics
+#
+[SERVICE]
+    flush           1
+    log_level       info
+
+[INPUT]
+    name            process_exporter_metrics
+    tag             process_metrics
+    scrape_interval 2
+
+[OUTPUT]
+    name            prometheus_exporter
+    match           process_metrics
+    host            0.0.0.0
+    port            2021
+```
+
+You can test the expose of the metrics by using _curl:_
+
+```bash
+curl http://127.0.0.1:2021/metrics
+```
+
+### Container to Collect Host Metrics
+
+When deploying Fluent Bit in a container you will need to specify additional settings to ensure that Fluent Bit has access to the host operating system. The following docker command deploys Fluent Bit with specific mount path for ptocfs and settings enabled to ensure that Fluent Bit can collect from the host. These are then exposed over port 2021.
+
+```
+docker run -ti -v /proc:/host/proc \
+               -v /sys:/host/sys   \
```

Comment:
Why do we need `/sys`? We have not mentioned it anywhere...

Reply:
Oh, I forgot to remove this...